Cesium3DTilesSelection/src/EllipsoidTilesetLoader.cpp

```diff
@@ -0,0 +1,279 @@
+#include <Cesium3DTilesContent/ImplicitTilingUtilities.h>
+#include <Cesium3DTilesSelection/EllipsoidTilesetLoader.h>
+#include <Cesium3DTilesSelection/Tile.h>
+#include <Cesium3DTilesSelection/TileContent.h>
+#include <Cesium3DTilesSelection/TileLoadResult.h>
+#include <Cesium3DTilesSelection/TileRefine.h>
+#include <Cesium3DTilesSelection/Tileset.h>
+#include <Cesium3DTilesSelection/TilesetContentLoader.h>
+#include <Cesium3DTilesSelection/TilesetExternals.h>
+#include <Cesium3DTilesSelection/TilesetOptions.h>
```

Comment:
There's a lot of unnecessary duplication between the headers included here and in the .h file. Some general rules:

* Always forward declare instead of including a header, if you can.
* Always make sure everything that is needed is included. Don't rely on someone else including some other header before they include yours.
* In general, a source file should include what it uses. But if a .h file includes a particular header because it has a field of that type, for example, it's not necessary to also include it in the .cpp file where that field is used.

We're a bit particular about this because it helps keep compile times as low as possible.